docs/guides/developer/docs/develop/setup-opencast-for-develop.md

```diff
@@ -89,155 +69,91 @@ install them.
-$ dnf group install 'Development Tools'
-$ dnf install -y ffmpeg maven tesseract hunspell sox synfig unzip gcc-c++ tar bzip2 nodejs
-```
-
-
-macOS 10.14
------------
+## Build Opencast
 
-### Update System
 
-Try to install all updates via the App Store or the Apple Icon on the top left corner.
+You can build now opencast by changing your directory into your opencast location and by running:
 
-### Java JDK 11
+    $ mvn clean install [Options]
 
-Install the JDK 11
+After the successful compilation you can start opencast with:
 
-### XCode
+    $ ./build/opencast-dist-develop-*/bin/start-opencast
 
-Install XCode over the App Store. It will be needed for building and for git.
+Where `*` is the version number of the distribution.
 
-### Install Packages via Homebrew
+### Build options
 
-The Homebrew Project adds a package manager to Mac OS. You can install it by:
+#### Assembly profiles
 
-```sh
-$ /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-```
+Besides the default `dist` Maven profile, you can define other profiles to create different builds.
+Activate these profiles using:
 
-You can now install needed packages:
+profile | description
+--------|------------
+`-Pdev`   | Create a single `allinone` distribution and unpack it
+`-Pnone`  | Do not create any distribution
 
-```sh
-$ brew install maven ffmpeg nodejs
-```
+You can only use one of these profiles at a time.
 
-### Git Bash Completion
+#### Build options
 
-In macOS you can not complete or suggest half typed commands with your Tab Key (like you probably know from linux). If you want to use bash completion, you have to install it by
+The options can be used to customize the build process. They can be used in combination with the assembly profiles.
 
-```sh
-$ brew install bash-completion
-```
-
-Find the location of the configuration file
+Option | Description
+-------|------------
+`-DtrimStackTrace=false` | Do not trim stack traces in the logs
+`-DskipTests=true` | Skip tests
+`-T 1.0C` | Use multiple threads for building (Experimental)
 
-```sh
-$ sudo find / -type f -name "git-completion.bash"
-```
+### Build Single Modules
 
-Normally it should be in
+When working on a single Opencast module, it can be extremely helpful to watch the newly built version and include
+it automatically in the Opencast OSGi infrastructure. This can be done through the
+[bundle:watch](https://karaf.apache.org/manual/latest/commands/bundle-watch.html) command in Karaf. The workflow would
+be as follows:
 
-    $ cp /Library/Developer/CommandLineTools/usr/share/git-core/git-completion.bash /usr/local/etc/bash_completion.d/
+* Start Opencast and use `la -u` in the Karaf console to list all installed bundles/modules. Note down the IDs of the
```

Comment:
New developers likely don't what "Karaf console" is and especially not how to access it. Maybe make that the first step?